pkg/crypto/bcrypt.go

```diff
@@ -0,0 +1,24 @@
+package crypto
+
+import (
+	"eniqilo-store/config"
+	"strconv"
+
+	"golang.org/x/crypto/bcrypt"
+)
+
+func GenerateHashedPassword(password string) (string, error) {
+	costStr := config.GetString("BCRYPT_SALT")
```

Comment:
agak kurang familiar buat get config by string gini. 
biasanya udah bentuk access object. 
klo string gini kan lebih rawan human error instead of access config object nya langsung

Reply:
oke mas, nanti coba ku improve di PR lain